x-pack/plugin/mapper-aggregate-metric/src/test/java/org/elasticsearch/xpack/aggregatemetric/mapper/AggregateDoubleMetricFieldMapperTests.java

```diff
@@ -165,6 +168,33 @@ protected List<ExampleMalformedValue> exampleMalformedValues() {
             // invalid metric value
             exampleMalformedValue(b -> b.startObject().field("min", "10.0").field("max", 50.0).field("value_count", 14).endObject())
                 .errorMatches("Failed to parse object: expecting token of type [VALUE_NUMBER] but found [VALUE_STRING]"),
+            // invalid metric value with additional data
```

Comment:
Maybe mention that the `field("hello", "world")` triggers the error, and we expect everything before and after to show up in synthetic source.